code/modules/modular_computers/file_system/programs/radar.dm

```diff
@@ -0,0 +1,295 @@
+			program_icon_state = "[initial(program_icon_state)]far"
+
+	if(last_icon_state != program_icon_state)
+		computer.update_icon()
+		last_icon_state = program_icon_state
+	computer.setDir(get_dir(here_turf, target_turf))
+
+//We can use process_tick to restart fast processing, since the computer will be running this constantly either way.
+/datum/computer_file/program/radar/process_tick()
+	if(computer.active_program == src)
+		START_PROCESSING(SSfastprocess, src)
+
+///////////////////
+//Suit Sensor App//
+///////////////////
+
+///A program that tracks crew members via suit sensors
+/datum/computer_file/program/radar/lifeline
+	filename = "lifeline"
+	filedesc = "Lifeline"
+	extended_desc = "This program allows for tracking of crew members via their suit sensors."
+	requires_ntnet = TRUE
+	transfer_access = ACCESS_MEDICAL
+	available_on_ntnet = TRUE
+
+/datum/computer_file/program/radar/lifeline/find_atom()
+	return locate(selected) in GLOB.carbon_list //currently we dont have a list of humanoids so this'll have to do
+
+/datum/computer_file/program/radar/lifeline/scan()
+	if(world.time < next_scan)
+		return
+	next_scan = world.time + (2 SECONDS)
+	objects = list()
+	for(var/i in GLOB.carbon_list)
+		var/mob/living/carbon/human/humanoid = i
+		if(!trackable(humanoid))
+			continue
+		var/crewmember_name = "Unknown"
+		if(humanoid.wear_id)
+			var/obj/item/card/id/ID = humanoid.wear_id.GetID()
+			if(ID?.registered_name)
+				crewmember_name = ID.registered_name
+		var/list/crewinfo = list(
+			ref = REF(humanoid),
+			name = crewmember_name,
+			)
+		objects += list(crewinfo)
+
+/datum/computer_file/program/radar/lifeline/trackable(mob/living/carbon/human/humanoid)
+	if(!humanoid || !istype(humanoid))
+		return FALSE
+	if(..())
+		if(humanoid in SSnanites.nanite_monitored_mobs)
+			if(humanoid.is_jammed())
+				return FALSE
+			return TRUE
+		if(istype(humanoid.w_uniform, /obj/item/clothing/under))
+			var/obj/item/clothing/under/uniform = humanoid.w_uniform
+			if(uniform.is_jammed())
+				return FALSE
+			if(uniform.has_sensor && uniform.sensor_mode >= SENSOR_COORDS) // Suit sensors must be on maximum
+				return TRUE
+	return FALSE
+
+////////////////////////
+//Nuke Disk Finder App//
+////////////////////////
+
+///A program that tracks nukes and nuclear accessories
+/datum/computer_file/program/radar/fission360
+	filename = "fission360"
+	filedesc = "Fission360"
+	program_icon_state = "radarsyndicate"
+	extended_desc = "This program allows for tracking of nuclear authorization disks and warheads."
+	requires_ntnet = FALSE
+	transfer_access = null
+	available_on_ntnet = FALSE
+	available_on_syndinet = TRUE
+	tgui_id = "NtosRadarSyndicate"
+	arrowstyle = "ntosradarpointerS.png"
+	pointercolor = "red"
+
+/datum/computer_file/program/radar/fission360/find_atom()
+	return locate(selected) in GLOB.poi_list
+
+/datum/computer_file/program/radar/fission360/scan()
+	if(world.time < next_scan)
+		return
+	next_scan = world.time + (2 SECONDS)
```

Comment:
Wouldn't it be better to use our cooldowns?

Reply:
what do you mean